features/login/impl/src/main/kotlin/io/element/android/features/login/impl/changeserver/ChangeServerView.kt

```diff
@@ -114,7 +112,7 @@ fun ChangeServerView(
         topBar = {
             TopAppBar(
                 title = {},
-                navigationIcon = { BackButton(onClick = onBackPressed, enabled = interactionEnabled) }
+                navigationIcon = { BackButton(onClick = onBackPressed, enabled = !isLoading) }
```

Comment:
I do not really like to block the user from going back, even if there is a loading in progress... I think the user should always be able to go back and in this case we should cancel the action.

But this is not new with this PR, so not blocking here.

Reply:
You're right, I'll remove it.